app/scripts/lib/snap-keyring/metrics.ts

```diff
@@ -13,7 +13,7 @@ export type SnapAndHardwareMessenger = RestrictedControllerMessenger<
   never
 >;
 
-export default async function getSnapAndHardwareInfoForMetrics(
+export async function getSnapAndHardwareInfoForMetrics(
```

Comment:
Just for audit sake, is this a merge error or have we fixed this because it was exporting a default and additional things like the type above?

Reply:
Really minor but I didn't want to mock mark with `__esModule: true` in `transaction-metrics.test.ts`. Also find it easier to see function name in the test file, rather than `default: jest.fn().mockImplementation...`